pkg/controller/operators/catalog/operator.go

```diff
@@ -890,7 +890,7 @@ func (o *Operator) syncResolvingNamespace(obj interface{}) error {
 	// get the set of sources that should be used for resolution and best-effort get their connections working
 	logger.Debug("resolving sources")
 
-	querier := resolver.NewNamespaceSourceQuerier(o.sources.AsClients(o.namespace, namespace))
```

Comment:
TIL: this wasn't (never?) actually dependency injected

even if it was, the DI should have taken a factory anyway